modules/ztp-image-based-upgrade-prep-oadp.adoc

```diff
@@ -72,65 +72,23 @@ The same version of the applications must function on both the current and the t
 ====
 --
 
-. Create the `oadp-cm` `ConfigMap` object through the `oadp-cm-policy` in a new `PolicyGenTemplate` called `ibu-upgrade-ranGen.yaml`:
-+
-[source,yaml]
-----
-apiVersion: ran.openshift.io/v1
-kind: PolicyGenTemplate
-metadata:
-  name: example-group-ibu
-  namespace: "ztp-group"
-spec:
-  bindingRules:
-    group-du-sno: ""
-  mcp: "master"
-  evaluationInterval: 
-    compliant: 10s
-    noncompliant: 10s
-  sourceFiles:
-  - fileName: ConfigMapGeneric.yaml
-    complianceType: mustonlyhave
-    policyName: "oadp-cm-policy"
-    metadata:
-      name: oadp-cm
-      namespace: openshift-adp
-----
-
 . Create a `kustomization.yaml` with the following content:
 +
 [source,yaml]
 ----
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-generators: <1>
-- ibu-upgrade-ranGen.yaml
-
-configMapGenerator: <2>
+configMapGenerator: <1>
 - files:
   - source-crs/ibu/PlatformBackupRestore.yaml
   #- source-crs/custom-crs/ApplicationClusterScopedBackupRestore.yaml
   #- source-crs/custom-crs/ApplicationApplicationBackupRestoreLso.yaml
   name: oadp-cm
   namespace: ztp-group
```

Comment:
Add a note for this field. Even though the cm is generated on the hub, for IBGU to copy it to the spoke, the ns must also exist on all the spokes